cylc/flow/install_plugins/log_vc_info.py

```diff
@@ -265,7 +263,8 @@ def write_vc_info(
     """
     if not info:
         raise ValueError("Nothing to write")
-    info_file = Path(run_dir, LOG_VERSION_DIR, INFO_FILENAME)
+    info_file = Path(
+        run_dir, WorkflowFiles.LogDir.VERSION, INFO_FILENAME)
```

Comment:
This isn't the same as the original path. We need `run_dir/log/version`, at the moment it is making `run_dir/version`.

Reply:
Thanks Mel - this has turned out to be a bigger refactor than intended